taxjar-ruby.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.0'
 
   spec.add_dependency 'addressable', '~> 2.3'
-  spec.add_dependency 'http', '~> 2.2'
+  spec.add_dependency 'http', '>= 1.0', '< 5.0'
```

Comment:
Is it worth locking the upper end of this to 4.0.3, in case a breaking change is introduced between 4.0.3 and <5.0? I know that shouldn't happen and is unlikely, but debugging gem issues can be a pain.

Reply:
Both major version bumps (3.0/4.0) were due to dropping Ruby support. I think <5.0 is reasonable, because there shouldn't be any breaking changes between 4.0.3 and <5.0.